runtime/coverage.go

```diff
@@ -203,31 +202,32 @@ func (r *CoverageReport) InspectProgram(location Location, program *ast.Program)
 		})
 
 	locationCoverage := NewLocationCoverage(lineHits)
-	r.Coverage[location] = locationCoverage
+	r.Coverage[location.ID()] = locationCoverage
 }
 
 // IsProgramInspected checks whether the *ast.Program on the given
 // location, has been inspected or not.
 func (r *CoverageReport) IsProgramInspected(location Location) bool {
-	_, isInspected := r.Programs[location]
+	_, isInspected := r.Programs[location.ID()]
 	return isInspected
 }
 
-// CoveredStatementsPercentage returns a string representation of
-// the covered statements percentage. It is defined as the ratio
-// of total covered lines over total statements, for all locations.
-func (r *CoverageReport) CoveredStatementsPercentage() string {
-	totalStatements := 0
-	totalCoveredLines := 0
-	for _, locationCoverage := range r.Coverage { // nolint:maprange
-		totalStatements += locationCoverage.Statements
-		totalCoveredLines += locationCoverage.CoveredLines()
-	}
-	percentage := fmt.Sprintf(
+// Percentage returns a string representation of the covered statements
+// percentage. It is defined as the ratio of total covered lines over
+// total statements, for all locations.
+func (r *CoverageReport) Percentage() string {
```

Comment:
This method could be utilized [here](https://github.com/onflow/flow-cli/pull/827/files#diff-b46e7eeb2d9c6cf9e1f6d217bb509d83d98f83bea1eaac7b7c99fcbf534fa261R132), instead of using the result from `CoveredStatementsPercentage`.